### PR TITLE
[Documentation] Updating CHANGELOG with missing content

### PR DESCRIPTION
**Pull request by JoshuaLai**
## Purpose
Updating CHANGELOG with missing content

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,15 +17,15 @@
 
 ### Bug Fixes
 - Cleaning up Calling State and Call Error when joining or returning to a call from error. [#28](https://github.com/Azure/communication-ui-library-ios/pull/28)
-- Fix the order of CompositeError throwing in joining a call. [#11](https://github.com/Azure/communication-ui-library-ios/pull/11)
-- Fix for info header doesn't disappear when bottom drawer is displayed. [#12](https://github.com/Azure/communication-ui-library-ios/pull/12)
-- Fix for the local video rotation in setup view when device orientation is locked. [#13](https://github.com/Azure/communication-ui-library-ios/pull/13)
-- Fix the display of the participant with an empty name in the participant list. [#24](https://github.com/Azure/communication-ui-library-ios/pull/24)
-- Fix the default audio selection when the UI Composite is launched. [#21](https://github.com/Azure/communication-ui-library-ios/pull/21)
+- Fixing the order of CompositeError throwing in joining a call. [#11](https://github.com/Azure/communication-ui-library-ios/pull/11)
```
Comment (vhuseinova-msft) [inline CHANGELOG.md:20]: Maybe we can use root form of verbs instead?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 ## 1.0.0-beta.3 (Upcoming)
 ### New Features
-- Update minimum iOS version to 14.0
+- Update minimum iOS version to 14.0. [#103](https://github.com/Azure/communication-ui-library-ios/pull/103)
 
 ## 1.0.0-beta.2 (2022-04-04)
-
 ### New Features
 - Update joining experience in setup view. [#44](https://github.com/Azure/communication-ui-library-ios/pull/44)
 - Support pinch & zoom, pan and double tap to zoom in screen share presentation. [#29](https://github.com/Azure/communication-ui-library-ios/pull/29)
```
Comment (vhuseinova-msft) [inline CHANGELOG.md:9]: `Implemented Localization support for 13 languages and right-to-left mirror layout. [#73](https://github.com/Azure/communication-ui-library-ios/pull/73) `

Should be updated too

Comment (JoshuaLai) [reply CHANGELOG.md:9]: This is already in the `changelog` on line 11 so no change needed.

File: CHANGELOG.md (hunk 1 above)
Comment (JoshuaLai) [reply CHANGELOG.md:20]: So are you recommending we we just change this to?
```suggestion
- Fix the order of CompositeError throwing in joining a call. [#11](https://github.com/Azure/communication-ui-library-ios/pull/11)
```

Comment (JoshuaLai) [reply CHANGELOG.md:20]: As per conversation changing to `fixed`

**Review by vhuseinova-msft — APPROVED** [review]

**Review by jimchou-dev — APPROVED** [review]

**Pull request merged**